fastify.js

```diff
@@ -224,6 +226,17 @@ function build (options) {
     }
 
     req.id = genReqId(req)
+
+    if (trustProxy) {
+      const proxyFn = getTrustProxyFn()
```

Comment:
can you move this to outside the routeHandler function?

Reply:
@mcollina Yes. Would it be better if I move `if` outside as well and if `hasTrustProxy` is not set it will use `noop` function?

Reply:
Yes. thant’s ok!